text/0230-telemetry-viewer.md

```diff
@@ -0,0 +1,129 @@
+# Telemetry Viewer for Developers
+
+_A local telemetry viewer to aid in instrumentation and debugging of pipelines._
```

Comment:
It can also be useful for application developers to run in their local environment while doing dev/test to ensure that the telemetry they are emitting is actually being exported.